modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-buildah-build.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * builds/uninstalling-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-buildah-build-in-a-network-restricted-environment_{context}']
+= Creating buildah build in a network restricted environment
+
+You can create a `buildah` build in a network network restricted environment by mirroring the images required by the `buildah` buildStrategy and then creating a `buildah` build. 
```

Comment:
```suggestion
You can create a `buildah` build in a network restricted environment by mirroring the images required by the `buildah` BuildStrategy and then creating a `buildah` build. 
```
There is a slight repetition of phrases (create a `buildah` build AND creating a `buildah` build) in this sentence. Not critical, but can we please reword it?